### DISCUSSION TITLE
Mensalidade 12 MESES

### DISCUSSION THREAD

**Discussion by iEmersonJr** [RamonSilva20/mapos#2310] [Q&A]
Saudações amigos,

Existe algum modo ou modificação do mapos gerar um faturamento de mensalidades de uma vez? 
Por exemplo: cliente precisa do serviço durante 12 meses. E ir quitando parcela por parcela conforme o pagamento é realizado. Iguais em provedores de internet, gera o financeiro anual e conforme o cliente utiliza vai quitando. 

Não sei se estou no topico certo, mas gostaria muito de saber se é possivel.

**Comment by GABRIEL-PI** [comment]
Se chama mult-tenant, ou mult empresas, ainda não existe uma versão do mapos assim, porem alguns ja modificaram.
